py/fiberassign/gfa.py

```diff
@@ -175,10 +176,8 @@ def get_gfa_targets(tiles, gfafile, faintlim=99, gaiadr="dr2"):
         # AR not passing the Gaia AEN criterion (PM correction done for AEN targets only)
         g = t["GAIA_PHOT_G_MEAN_MAG"]
         aen = t["GAIA_ASTROMETRIC_EXCESS_NOISE"]
-        isaen = np.logical_or(
-            (g <= 19.0) * (aen < 10.0 ** 0.5),
-            (g >= 19.0) * (aen < 10.0 ** (0.5 + 0.2 * (g - 19.0))),
-        )
+        gaiadr = "dr2" if "G2" in t["REF_CAT"] else "edr3"
```

Comment:
Provided this is only ever used for a full `gfas-*` file, it will work. There are possibly strange cases where somebody passes a single object and it happens to be a Tycho (`T2`) object, but you might not want to bother catching those cases? It's up to you!